app/src/main/res/values/strings.xml

```diff
@@ -24,12 +24,12 @@
     <string name="home_notice_content">Download Magisk ONLY from the official GitHub page. Files from unknown sources can be malicious!</string>
     <string name="home_support_title">Support Us</string>
     <string name="home_item_source">Source</string>
-    <string name="home_support_content">Magisk is, and always will be, free and open-source. You can however show us that you care by sending a small donation.</string>
+    <string name="home_support_content">Magisk will always be copylefted libre software.\nUse,study, change and share; with all.\nShow you care by sending a donation.</string>
     <string name="home_installed_version">Installed</string>
     <string name="home_latest_version">Latest</string>
     <string name="invalid_update_channel">Invalid Update Channel</string>
     <string name="uninstall_magisk_title">Uninstall Magisk</string>
-    <string name="uninstall_magisk_msg">All modules will be disabled/removed!\nRoot will be removed!\nYour data potentially encrypted if not already!</string>
```

Comment:
This is broken English.
"Your data potentially encrypted" means the data belonging to the user possibly encrypted something.

Reply:
I disagree, as a Canadian English speaker this makes perfect sense to me.

Reply:
@osm0sis 
Then what does that sentence mean?
Could you ask someone to explain it to you and then either give up or enlighten me, because I really can't work it out.

Reply:
Because any force decryption flags Magisk may have added to the fstab will be reverted any device using these to run decrypted will potentially be encrypted again.

Reply:
Aha. Semantically I understand: "Your data will potentially be encrypted, if it is not already!" to mean that.

Reply:
IMHO, the only changes needed to make this sentence more understandable are:
- "disabled/removed" -> "disabled and removed"
- "Your data potentially encrypted if not already!" -> "Your data will potentially be encrypted, if it is not already!"

Anything else is unnecessary and has the potential to reduce clarity.
As it is now, it looks much more confusing than the original ("Storages decrypted in Magisk" doesn't mean anything really).

Reply:
You are saying "potential to reduce clarity" and "more confusing", but I don't know why and there is no explanation.
It could very well be that Magisk isn't for me, but I make good use of it.

"Storages decrypted in Magisk" means any storage that was decrypted in Magisk (presumably by the user), understood as  ~since it was installed."
It could be "Storages you used Magisk to decrypt will be re-encrypted". Is that better? It assumes Magisk isn't decrypting volumes on its own.

I can't understand the original to mean changed mount options, because
"Your data will potentially be encrypted, if it is not already!" could also mean:

* All my data could be encrypted, potentially, since it isn't already. This seems like a risk of uninstalling Magisk because Magisk cannot guarantee whether encryption will occur or not.
* I have nothing to worry about, because if all my data is encrypted, there is no potential for it to be, since it _is_ already.
* I could understand data to mean the stuff I handle ~user data, like notes encrypted by note apps, as opposed to system files and the like.

Reply:
"turned of" != "turned off", make it say "disabled" again there, less ambiguity.

Reply:
A technical question on this string: when you say "storage devices", you mean device partitions?
Because I think it's pretty difficult to translate "storage devices" into Italian without risking to mislead the user. An external microSD is a storage device too, and after reading what @osm0sis said above _(any force encryption flags Magisk may have removed from the fstab)_, I understand that this may impact only internal device partitions.

Also, two nits now that the situation has settled a bit:
- _Root will be removed_ -> _Root permissions will be lost_ (root is not something you "remove")
- _through the use of Magisk_ -> _by Magisk_ (as it is now it seems to imply that the action of "using root" decrypts the data, which is not accurate to say)

Reply:
Yeah, let's make it say internal storage to be clear.